webrtc/pending-promise-after-close.html

```diff
@@ -0,0 +1,81 @@
+<!doctype html>
+<title>Test pending promises behavior when RTCPeerConnection object is closed</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script type="text/javascript">
+  'use strict';
+
+  function test_never_resolved(test_func, testName) {
```

Comment:
Will fix. Was switching between the case convention of testharness and normal JS and got confused.